prime-router/src/test/kotlin/fhirengine/engine/CustomFhirPathFunctionTest.kt

```diff
@@ -150,4 +155,30 @@ class CustomFhirPathFunctionTest {
             (result[0] as StringType).value
         ).isEqualTo(loincCode)
     }
+
```

Comment:
Need to include tests for sadpath as well as happypath (i.e. what happens when value not found)